ui/components/app/assets/nfts/nft-details/nft-details.tsx

```diff
@@ -110,9 +108,10 @@ export default function NftDetails({ nft }: { nft: Nft }) {
 
   const nftImageAlt = getNftImageAlt(nft);
   const nftSrcUrl = imageOriginal ?? image;
-  const nftImageURL = getAssetImageURL(imageOriginal ?? image, ipfsGateway);
   const isIpfsURL = nftSrcUrl?.startsWith('ipfs:');
-  const isImageHosted = image?.startsWith('https:');
+  const isImageHosted =
+    image?.startsWith('https:') || image?.startsWith('http:');
```

Comment:
I usually like to use the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol) constructor for this sort of thing, but no big deal

Reply:
resolving this for now, because we discussed that doing this change would also involve adding error handling, 
Because this change is not critical to this upgrade, we will revisit in future 🙏